src/main/resources/ValidationMessages.properties

```
@@ -2,4 +2,14 @@ autor.nome.obrigatorio = O nome é obrigatório
 autor.email.valido = O email é obrigatório e deve ter o formato válido
 autor.descricao.maximo = A descrição não pode conter mais que 400 caracteres
 autor.descricao.obrigatorio = A descrição é obrigatória
-categoria.nome.obrigatorio = O nome é obrigatório
\ No newline at end of file
+categoria.nome.obrigatorio = O nome é obrigatório
+livro.titulo.obrigatorio = O título é obrigatório
+livro.resumo.maximo = O resumo deve conter até 500 caracteres
+livro.resumo.obrigatorio = O resumo é obrigatório
+livro.sumario.obrigatorio = O sumário é obrigatório
+livro.preco.minimo = O valor mínimo é de R$20,00
+livro.paginas.minimo = O livro deve ter ao menos 100 páginas
```

Aqui, é melhor deixar menos fixo, pegando o valor definido lá na anotação da seguinte maneira:

```suggestion
livro.preco.minimo = O valor mínimo é de R$ {value}
livro.paginas.minimo = O livro deve ter ao menos {value} páginas
```

O mesmo deve ser feito para a mensagem de `autor.descricao.maximo`.